wazimap/views.py

```diff
@@ -33,6 +33,7 @@ class HomepageView(TemplateView):
     def get_context_data(self, *args, **kwargs):
         return {
             'root_geo': geo_data.root_geography(),
+            'video_links': settings.WAZIMAP.get('video_links', None)
```

Comment:
I think that this is an indication that what we're doing isn't ready for the wazimap distribution yet. The WAZIMAP dict is injected into all wazimap views (that's where WAZIMAP.name comes from), so we don't need this.